operations/patch.go

```diff
@@ -131,7 +131,6 @@ func Patch() cli.Command {
 				SkipConfirm:       c.Bool(skipConfirmFlagName) || outputJSON,
 				Description:       c.String(patchDescriptionFlagName),
 				AutoDescription:   c.Bool(autoDescriptionFlag),
-				Finalize:          c.Bool(patchFinalizeFlagName),
```

Comment:
since we removed this, should we update this function to always return false and explain why? 

https://github.com/evergreen-ci/evergreen/blob/7606fc299cbd57bd4026b95a603c3bd8871fcf28/model/patch/cli_intent.go#L171

Reply:
ah good catch we might just be able to remove that field

Reply:
This will need to be done in a followup ticket since changing this server-side function requires that everyone has upgraded their CLI to include this change.

Reply:
Filed [DEVPROD-8213](https://jira.mongodb.org/browse/DEVPROD-8213)

Reply:
@hadjri are we talking about the same thing? Why would users need to update to include this change? We aren't getting rid of the finalize flag, we're just getting rid of the cliIntent field, since we aren't using it. Users can still use the finalize flag to finalize their patch, we just aren't using the cliIntent field to do it.